polaris-core/src/main/java/org/apache/polaris/core/persistence/resolver/PolarisResolutionManifest.java

```diff
@@ -68,6 +68,14 @@ public class PolarisResolutionManifest implements PolarisResolutionManifestCatal
   // Set when resolveAll is called
   private ResolverStatus primaryResolverStatus = null;
 
+  private ResolverStatus requirePrimaryResolverStatus() {
```

Comment:
minor : require gives an impression that its just doing assertion, but its additionally returning status how about this 

```suggestion
  private ResolverStatus safeGetPrimaryResolverStatus() {
```